packages/cli/src/cmds/assert/assert.ts

```diff
@@ -0,0 +1,77 @@
+import process from 'process';
+import { verbose } from '../../utils';
+import ValidationError from '../error/validationError';
+import AbortError from '../error/abortError';
+import * as fs from 'fs';
+import 'fs/promises';
+import * as console from 'console';
+import { buildAppMap } from '../../search/utils';
+import AssertionChecker from './assertionChecker';
+import Assertion from './assertion';
+
+
+exports.command = 'assert <appMapsDir>';
+exports.describe = 'Run assertions for AppMaps in the directory';
+
+exports.builder = (args) => {
+  args.positional('appMapsDir', {
```

Comment:
Other commands use: ```
      args.option('appmap-dir', {
        describe: 'directory to recursively inspect for AppMaps',
        default: 'tmp/appmap',
      });
```

Reply:
Changed in 2bbe5f902fdfa10d8e47c7dd76c8f91f401aaf4c.